docs/content/latest/secure/authentication/client-authentication.md

```diff
@@ -129,17 +129,17 @@ Specifies the client machine addresses that this record matches. This field can
 
 You can also write `all` to match any IP address, `samehost` to match any of the server's own IP addresses, or `samenet` to match any address in any subnet that the server is directly connected to.
 
-If a host name is specified (anything that is not an IP address range or a special key word is treated as a host name), that name is compared with the result of a reverse name resolution of the client's IP address (e.g., reverse DNS lookup, if DNS is used). Host name comparisons are case insensitive. If there is a match, then a forward name resolution (e.g., forward DNS lookup) is performed on the host name to check whether any of the addresses it resolves to are equal to the client's IP address. If both directions match, then the entry is considered to match. (The host name that is specified in the `--pg_hba_conf` flag should be the one that address-to-name resolution of the client's IP address returns, otherwise the line won't be matched. Some host name databases allow associating an IP address with multiple host names, but the operating system will only return one host name when asked to resolve an IP address.)
+If a host name is specified (anything that is not an IP address range or a special key word is treated as a host name), that name is compared with the result of a reverse name resolution of the client's IP address (e.g., reverse DNS lookup, if DNS is used). Host name comparisons are case-insensitive. If there is a match, then a forward name resolution (e.g., forward DNS lookup) is performed on the host name to check whether any of the addresses it resolves to are equal to the client's IP address. If both directions match, then the entry is considered to match. (The host name specified in the `--pg_hba_conf` flag should be the one that address-to-name resolution of the client's IP address returns, otherwise the line won't be matched. Some host name databases allow associating an IP address with multiple host names, but the operating system will only return one host name when asked to resolve an IP address.)
 
 A host name specification that starts with a dot (`.`) matches a suffix of the actual host name. So `.example.com` would match `foo.example.com` (but not just `example.com`).
 
-When host names are specified using the `--pg_hba_conf` flag, you should make sure that name resolution is reasonably fast. It can be of advantage to set up a local name resolution cache such as `nscd`. Also, you may wish to enable the configuration parameter `log_hostname` to see the client's host name instead of the IP address in the log.
+When host names are specified using the `--pg_hba_conf` flag, you should make sure that name resolution is reasonably fast. It can be advantageous to set up a local name resolution cache, such as `nscd`. Also, you may want to enable the configuration parameter `log_hostname` to see the client's host name instead of the IP address in the log.
```

Comment:
```suggestion
When host names are specified using the `--ysql_hba_conf` flag, you should make sure that name resolution is reasonably fast. It can be advantageous to set up a local name resolution cache, such as `nscd`. Also, you may want to enable the configuration parameter `log_hostname` to see the client's host name instead of the IP address in the log.
```